### PR TITLE
Getting Started Guide

### PR DESCRIPTION
**Pull request by TChag**
Quick Git guide using Git Desktop

### CODE REVIEW

File: GETTIN STARTED.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Simple Instructions for Git Collaboration
+
+SetUp
```
Comment (gordonbrander) [inline GETTIN STARTED.md]: Nit: should be "Setup" (no camel-case).

File: GETTIN STARTED.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Simple Instructions for Git Collaboration
```
Comment (gordonbrander) [inline GETTIN STARTED.md]: Can we make this an h1? (`# Simple Instructions for Git Collaboration`)

File: GETTIN STARTED.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+SetUp
+
+1. Set up a Github account: https://github.com/
+2. Download Github Desktop (Mac or Windows): https://desktop.github.com/
+3. Authenticate to Github:  https://help.github.com/desktop/guides/getting-started/authenticating-to-github
+
+Usage
```
Comment (gordonbrander) [inline GETTIN STARTED.md]: Can we make these section headers h2 (`## Usage`)?

File: README.md [hunk 4]
```diff
@@ -50,3 +50,5 @@ Status
 - [ ] Download recipe
 - [ ] Sync dataset
 - [ ] List recipes
+
+Tin - quick test
```
Comment (gordonbrander) [inline README.md]: Can we remove these lines?

**Comment by TChag** [comment]
Formatting changes and update to Readme.

File: GETTIN STARTED.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+#Simple Instructions for Git Collaboration
```
Comment (gordonbrander) [inline GETTIN STARTED.md]: ^ Typo: file name should be GETTING_STARTED.md :)

File: GETTIN STARTED.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+##SetUp
```
Comment (gordonbrander) [inline GETTIN STARTED.md]: Can we make this "Setup"?

**Comment by TChag** [comment]
Rename and spelling error in Getting Started guide.

**Comment by gordonbrander** [comment]
Thanks 👍

**Pull request merged**